install/bin/editPasswd

```diff
@@ -1,2 +1,6 @@
+#!/bin/bash
+SCRIPT=$(readlink -f "$0")
+# Absolute path this script is in, thus /home/user/bin
+APP_PATH=$(dirname "$SCRIPT")
 
-%INSTALL_PATH/bin/decj ilex.util.PasswordFileEditor -f $LRGSHOME/.lrgs.passwd
+${APP_PATH}/decj ilex.util.PasswordFileEditor -f $LRGSHOME/.lrgs.passwd
```

Comment:
Should we handle checking for $LRGSHOME like done in ./elp below.

Reply:
Yep, good catch.